### DISCUSSION TITLE
ClusterWidget: port /pc104 is not responding

### DISCUSSION THREAD

**Discussion by 0707yiliu** [robotology/community#417] [Q&A] [answered]
**Describe the bug**
We have configured the laptop and icub server, but the yarpmanager cannot connect to the port /pc104

**Expected behavior**
We can ping to the pc104 and ssh to it, but when use yarpmanger connect the pc104, the yarprun is useless.
I just configure the ssh.

**Screenshots**
[WARNING]Cannot connect to /pc104 
[ERROR]ClusterWidget: port /pc104 is not responding 

**Configuration (please complete the following information):**
 - OS:Ubuntu18.04.4
 - yarp version:3.2.0
-  compiler:cmake3.17

**Comment by pattacini** [comment, marked as answer]
Hi @tKsome 

Did you already check whether launching `yarprun` manually works?
Do simply the following:
1. ssh into `pc104`
1. launch `yarprun --server /pc104`
1. try again and check if the yarpun broker is now alive (green light) from within the `yarpmanager`

**Comment by pattacini** [comment]
Closing, feel free to open it up again if needed.
